<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_DatabaseConnectionsConfigurationDialog</class>
 <widget class="QDialog" name="w_DatabaseConnectionsConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia - Baza danych</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>630</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="authDbConnectionGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>310</width>
     <height>280</height>
    </rect>
   </property>
   <property name="title">
    <string>Authorization Database</string>
   </property>
   <widget class="w_DatabaseConnectionWidget" name="AuthDbConnectionWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>300</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ClinicDbConnectionGroupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>310</width>
     <height>280</height>
    </rect>
   </property>
   <property name="title">
    <string>Clinic Database</string>
   </property>
   <widget class="w_DatabaseConnectionWidget" name="ClinicDbConnectionWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>300</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>w_DatabaseConnectionWidget</class>
   <extends>QWidget</extends>
   <header>w_databaseconnectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>w_DatabaseConnectionsConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>w_DatabaseConnectionsConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
